content/en/docs/kubernetes/operator/troubleshooting/target-allocator.md

```diff
@@ -0,0 +1,486 @@
+having the label `app: my-app`, which we see in the previous example.
+
+If your ServiceMonitor resource is missing that label, then the Target Allocator
+won’t pick up that ServiceMonitor.
+
+{{% alert title="Tip" %}}
+
+The same applies if you’re using a
+[PodMonitor](https://prometheus-operator.dev/docs/user-guides/getting-started/#using-podmonitors).
+In that case, if your `OpenTelemetryCollector`resource defined a
+[`podMonitorSelector`](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opentelemetrycollectorspectargetallocatorprometheuscr),
+then any `PodMonitors` you wish to be picked up by the Target Allocator would
+need to have that same label.
+
+{{% /alert %}}
+
+### Did you leave out the serviceMonitorSelector and/or podMonitorSelector configuration altogether?
+
+As we saw in
+["Did you configure a ServiceMonitor or PodMonitor selector"](#did-you-configure-a-servicemonitor-or-podmonitor-selector),
+setting mismatched values for `serviceMonitorSelector` and `podMonitorSelector`
+results in your `ServiceMonitors` and `PodMonitors`, respectively, not getting
+picked up.
+
+Similarly, in
+[`v1beta1`](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opentelemetrycollector-1)
+of the `OpenTelemetryCollector` CR, leaving out this configuration altogether,
+your `PodMonitors` and `ServiceMonitors` might also not get picked up.
+
+As of `v1beta1` of the `OpenTelemetryOperator`, a `serviceMonitorSelector`, and
+`podMonitorSelector` must be included, even if you don’t intend to use it, like
+this:
+
+```yaml
+---
+prometheusCR:
+  enabled: true
+  podMonitorSelector: {}
+  serviceMonitorSelector: {}
+```
+
+See the
+[full OpenTelemetryCollector definition in "Do you know if metrics are actually being scraped?"](#do-you-know-if-metrics-are-actually-beingscraped).
+
+### Do your labels, namespaces, and ports match for your ServiceMonitor and your Service (or PodMonitor and your Pod)?
+
+The `ServiceMonitor` is configured to pick up Kubernetes
+[Services](https://kubernetes.io/docs/concepts/services-networking/service/)
+that match on:
+
+- Labels
+- Namespaces (optional)
+- Ports (endpoints)
+
+Suppose that you have this `ServiceMonitor`:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: sm-example
+  labels:
+    app: my-app
+    release: prometheus
+spec:
+  selector:
+    matchLabels:
+      app: my-app
+  namespaceSelector:
+    matchNames:
+      - opentelemetry
+  endpoints:
+    - port: prom
+      path: /metrics
+    - port: py-client-port
+      interval: 15s
+    - port: py-server-port
+```
+
+The previous `ServiceMonitor` is looking for any services that have:
+
+- the label `app: my-app`
+- reside in a namespace called `opentelemetry`
+- a port named `prom`, `py-client-port`, _or_ `py-server-port`
+
+So for example, the `Service` resource below would get picked up by the
+`ServiceMonitor`, because it matches the above criteria:
```

Comment:
```suggestion
For example, the following `Service` resource would get picked up by the
`ServiceMonitor`, because it matches the previous criteria:
```